### ISSUE TITLE
Question: Grommet and create-react-app

### ISSUE THREAD

**Issue by luebken** [grommet/grommet#1509]
Hi

is this https://github.com/mvanlonden/grommet-css still the canonical to get Grommet running on top of a create-react-app? As a newbie it's hard to follow the discussion.

Matthias

**Comment by nomadus** [comment]
Good that you raised this issue, I also would like to validate the approach I have taken. 
There is a section in the Readme of CRA which talks about using css preprocessors, https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-a-css-preprocessor-sass-less-etc
What I did is created a custom scss file see as an example https://github.com/zooniverse/zoo-grommet/tree/master/src and run npm run build-css which produced a css file. 
It works OK so far, would this approach valid from the grommet point of view?

**Comment by karatechops** [comment]
There's a few approaches to getting Grommet working with create-react-app. One can be seen here in our [grommet-react-app](https://github.com/grommet/grommet-react-app), in this repo CRA is being ejected and all the sass loaders for Grommet are being put in place to run Grommet. As a newbie you probably want to steer clear of an ejected create-react-app as the Webpack stuff can be mind boggling.

The alternative is just pulling in Grommet's minified CSS bundle. If you look at the [Grommet distribution bundle](https://github.com/grommet/grommet/tree/stable) you'll see css files for each theme. In your root component you can simply import the css - `import 'grommet/grommet.min.css'`.

I put together a quick repo for you which uses the second method I described - https://github.com/karatechops/create-react-app-grommet

**Comment by karatechops** [comment]
I'm going to close this for now in an effort to keeps Grommet's backlog clean. Please feel free to keep the conversation going. I'm happy to continue assisting here as I know how confusing the Javascript tooling process can be. Also feel free to join us on [Slack](http://slackin.grommet.io/)!

**Comment by mjsevilla** [comment]
@karatechops are there any other major drawbacks to using your create-react-app-grommet repo? Also, sounds like we'll be able to correctly/elegantly use grommet w/ create-react-app when v2 comes out, yes?

**Comment by karatechops** [comment]
@mjsevilla the package dependencies need to be updated other than that it should be fine. V2 will seamlessly align with CRA as it's built on styled-components instead of Sass. With the create-react-app-grommet repo I'm simply importing Grommet's bundled css to the `index.js` file to retain grommet's styling information. I haven't built anything for production with the repo but in theory it should work 😄
